src/Tutorial/Tutorial.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Tutorial;
+
+use Hyperdrive\Combat\Combat;
+use Hyperdrive\Combat\Enemies;
+use Hyperdrive\Output\OutputContract;
+use Hyperdrive\Pilot\Pilot;
+use Hyperdrive\Ship\Ship;
+
+class Tutorial
+{
+
```

Comment:
Masz sporo pustych lini, często w niepotrzebnych miescach. Myślę, że warto zwracać na to uwagę, a kod stanie się czytelniejszy

Reply:
Starałem się na bieżąco formatować wszystko skrótem klawiszowym ctrl + alt + L , widocznie nie zauważyłem, że nie pousuwał on niektórych linii, które założyłem, że usunie.